tests/test_block.py

```diff
@@ -8,123 +8,134 @@
 
 class TestBlock(unittest.TestCase):
 
-        def test_block(self):
-            b = Block('name', 'type')
-            self.assertEqual(b.name, 'name')
-            self.assertEqual(b.type, 'type')
-
-        def test_noname(self):
-            with self.assertRaises(ValueError):
-                Block('', 'type')
-
-        def test_save(self):
-            b = Block('name', 'type', config)
-            b._instance = mock_instance()
-            b._put = MagicMock()
+    def test_block(self):
+        b = Block('name', 'type')
```

Comment:
All of these changes are in fcfeda8f6862ad52e10d5bccf85fc3cf2858f5cd

The indentation was off